packages/styles/scss/components/slider/_slider.scss

```diff
@@ -0,0 +1,230 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Slider
+//-----------------------------
+
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/skeleton' as *;
+@use '../../utilities/high-contrast-mode' as *;
+
```

Comment:
I think the main reason the styles are off is that we are using the `rem` function, but it's not being imported. Sass functions that aren't found just fail silently, instead of giving us an error (which is annoying!) 

`@use '../../utilities/convert' as *;`